benchflow-dsl/src/test/resources/tests/exploration/users/benchflow-test.yml

```diff
@@ -0,0 +1,90 @@
+###############################################################################
+# BenchFlow Test Definition
+###############################################################################
+version: '1'
+name: WfMSTest
+description: A WfMS test
+
+###############################################################################
+# Test Configuration
+###############################################################################
+configuration:
+  goal:
+    type: load
+
+    exploration_space:
+      workload:
+        users:
+          values: [5, 10, 15, 20]
+
+
+  users: 10
```

Comment:
Why do you have the setting also here, if you define an exploration space on users?